modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/components/select-objects/SelectObjects.es.js

```diff
@@ -76,44 +80,6 @@ export default ({defaultValue, label, onSelect, selectedValue, visible}) => {
 			});
 	};
 
-	const getMessageWithNewObjectLink = (message) => (
```

Comment:
Do we no longer need the link to the new object?

Reply:
No, we don't need. Now we can create a new object by the addButton inside the dropdown, and the link solution was temporary until we implement the current story.